apps/client/src/locales/messages.de.xlf

```diff
@@ -3481,6 +3481,22 @@
           <context context-type="linenumber">2</context>
         </context-group>
       </trans-unit>
+      <trans-unit id="46d3a0f17b741c93f9e61aa7157820da41506f53" datatype="html">
+        <source>Target Net Worth</source>
+        <target state="new">Angestrebtes Nettovermögen</target>
```

Comment:
😂 

You should also mark `state="translated"`

Reply:
updated :-)